src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/directionissued/DirectionIssuedAboutToSubmitHandlerTest.java

```diff
@@ -710,4 +710,54 @@ private void assertValues(PreSubmitCallbackResponse<SscsCaseData> response, Inte
         assertThat(response.getData().getDateSentToDwp(), is(LocalDate.now().toString()));
         assertThat(response.getData().getDwpDueDate(), is(LocalDate.now().plusDays(expectedResponseDays).toString()));
     }
+
+    @Test
+    public void givenNoUploadedAndGeneratedDoc_thenReturnErrorAndThenReturnCaseDataPreviewDocWhenGenerateNoticeIsYes() {
+        sscsCaseData.getDocumentGeneration().setGenerateNotice(NO);
```

Comment:
test description says generationNotice is Yes

Reply:
this was a bit tricky - since it was for the aboutToSubmitHandler, previewDoc would be set to null after its submitted, so there wasn't a way to check if it was capturing the correct doc. After it being submitted, it would start over and then next round take what was set in sscsCaseData (generateNotice = Yes) so we can check from here PreviewDoc would not be null if a doc was passed through